### ISSUE TITLE
pkgConfig template is generating prompts to email maintainers despite urls being declared

### ISSUE THREAD

**Issue by tfoote** [ros/catkin#878]
The pkgConfig.cmake.in falls back to maintainer name and email if there's an error even if there is a valid url declared.

https://github.com/ros/catkin/blob/kinetic-devel/cmake/templates/pkgConfig.cmake.in#L102

However projects with [valid url tags](https://github.com/ros/geometry/blob/indigo-devel/eigen_conversions/package.xml#L16) declared are generating these errors: 
```
-- ==> add_subdirectory(image_pipeline/depth_image_proc)
CMake Error at /home/matt/catkin_ws/devel/share/eigen_conversions/cmake/eigen_conversionsConfig.cmake:106 (message):
  Project 'eigen_conversions' specifies
  '/home/matt/catkin_ws/build_isolated/orocos_kdl/devel/../../include' as an
  include dir, which is not found.  It does neither exist as an absolute
  directory nor in
  '/home/matt/catkin_ws/src/geometry/eigen_conversions//home/matt/catkin_ws/build_isolated/orocos_kdl/devel/../../include'.
  Ask the maintainer 'Tully Foote <tfoote@osrfoundation.org>' to fix it.
Call Stack (most recent call first):
  /opt/ros/indigo/share/catkin/cmake/catkinConfig.cmake:76 (find_package)
  image_pipeline/depth_image_proc/CMakeLists.txt:4 (find_package)
```

I checked that catkin_pkg appears to be parsing correctly and defaulting the website value.
```
$ python cmake/parse_package_xml.py ~/work/github/ros/geometry/eigen_conversions/package.xml /tmp/out
tfoote@snowman:~/work/github/ros/catkin Last: [0] (0s Seconds)
$ cat /tmp/out 
set(_CATKIN_CURRENT_PACKAGE "eigen_conversions")
set(eigen_conversions_VERSION "1.11.8")
set(eigen_conversions_MAINTAINER "Tully Foote <tfoote@osrfoundation.org>")
set(eigen_conversions_PACKAGE_FORMAT "1")
set(eigen_conversions_BUILD_DEPENDS "cmake_modules" "geometry_msgs" "eigen" "orocos_kdl" "std_msgs")
set(eigen_conversions_BUILD_EXPORT_DEPENDS "geometry_msgs" "eigen" "orocos_kdl" "std_msgs")
set(eigen_conversions_BUILDTOOL_DEPENDS "catkin")
set(eigen_conversions_BUILDTOOL_EXPORT_DEPENDS )
set(eigen_conversions_EXEC_DEPENDS "geometry_msgs" "eigen" "orocos_kdl" "std_msgs")
set(eigen_conversions_RUN_DEPENDS "geometry_msgs" "eigen" "orocos_kdl" "std_msgs")
set(eigen_conversions_TEST_DEPENDS )
set(eigen_conversions_DOC_DEPENDS )
set(eigen_conversions_URL_WEBSITE "http://ros.org/wiki/eigen_conversions")
set(eigen_conversions_URL_BUGTRACKER "")
set(eigen_conversions_URL_REPOSITORY "")
set(eigen_conversions_DEPRECATED "")
```

**Comment by dirk-thomas** [comment]
I tried building `eigen_conversions` and the generated CMake config file selects the message with the website. Please provide a reproducible example which leads to your error message.

**Comment by tfoote** [comment]
The above string came in from an external user. 

And this is what's in the debians: 

```
tfoote@snowman:~ Last: [0] (0s Seconds)
$ cat /opt/ros/kinetic/share/eigen_conversions/cmake/eigen_conversionsConfig.cmake | grep tfoote
        message(FATAL_ERROR "Project 'eigen_conversions' specifies '${idir}' as an include dir, which is not found.  It does not exist in '${include}'.  Ask the maintainer 'Tully Foote <tfoote@osrfoundation.org>' to fix it.")
      message(FATAL_ERROR "Project 'eigen_conversions' specifies '${idir}' as an include dir, which is not found.  It does neither exist as an absolute directory nor in '/opt/ros/kinetic/${idir}'.  Ask the maintainer 'Tully Foote <tfoote@osrfoundation.org>' to fix it.")
```

But it looks like this has been resolved in #861 but it will only propagate when packages are rereleased with the next released version of catkin.
